content/docs/concepts/oracle-server/_index.md

```diff
@@ -0,0 +1,87 @@
+
+---
+title: "Autonity Oracle Server (AOS)"
+linkTitle: "Oracle Server"
+weight: 4
+description: >
+  Oracle server software features
+---
+
+AOS is the reference implementation of the Autonity Oracle Protocol and the oracle server software run by validator nodes in an Autonity network. It is composed of off- and on-chain components.
+
+## Features
+
+### Core logic
+
+- _Core_, the core off-chain Autonity Oracle Server (AOS) codebase. Core manages interactions with external price data source providers via data adaptor 'plugins' and the connected Autonity Go Client (AGC) validator node AOS serves. Core executes aggregation of data from external sources ("_L1 aggregation_"), calculates an aggregated median price, and submits price report transactions on-chain to the Oracle Contract.
+- _Autonity Oracle Contract_, the oracle protocol contract logic deployed to the ledger by AGC. The Oracle Contract manages the computation of median price data for currency pair price reports submitted by oracle servers. The contract provides operations for: computing median price ("_L2 aggregation_") from submitted price report transactions in oracle voting rounds, providing median price data via interface, and managing the currency-pair symbols for which price data is provided by the Autonity oracle network. See [Autonity Oracle Contract](/concepts/architecture/#autonity-oracle-contract) and concept [Client](/concepts/client/).
+- _Networking_, the system uses WebSocket and HTTP connections. HTTP RPC calls are made to configured data source providers, and a WebSocket connection is established to the AGC validator to (a) submit price report transactions and (b) listen for on-chain Oracle Contract events.
```

Comment:
The RPC calls to configured data source might have different network protocols, it may have HTTP, HTTPS, or event Web socket, it depends on the provider's scheme.

Reply:
Thanks! Edited to clarify in commit [3b4b466](https://github.com/autonity/docs.autonity.org/pull/75/commits/3b4b466b587da4cd3507a0ffbf068f0b9191b6e8).